src/routes/StackView/StackView.scss

```diff
@@ -22,6 +18,32 @@ $stack-view__parent-note-height: 22%;
   width: fit-content;
 }
 
+.stack-view__border {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  top: 0;
+  left: 0;
+  pointer-events: none;
+  background: transparent;
+  box-shadow: inset 0 0 0 $column__border-width var(--accent-color);
+}
+
+.stack-view__border--presenting {
```

Comment:
i'd prefer `moderating` over presenting to make it detectable by searching "modera..." (just so it has the same word stem as moderation), not that we introduce multiple words for the same feature

Reply:
you are also welcome to change everything to presenting instead of moderating (to not have it mixed up with moderating (as in "you are a moderator"))